index.html

```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
             </ul>
           </div>
         </div>
-
+        <hr />
         <div class="links">
           <a><i class="fa fa-github"></i></a>
           <a><i class="fa fa-twitter"></i></a>
           <a><i class="fa fa-envelope"></i></a>
           <a><i class="fa fa-linkedin"></i></a>
         </div>
+      </section>
+
+      <section class="works projects">
+        <span class="works1"></span>
+        <span class="works2"></span>
+        <div class="down">
+          <i class="fa fa-light fa-arrow-down"></i><span>Work</span>
+        </div>
+        <h3>MY RECENT</h3>
+        <h3>WORK</h3>
         <hr />
-        <p class="mail">joelkalema63@gmail.com</p>
+        <div class="project">
+          <div class="img">
+            <div class="picture"></div>
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="langage-used">
+            <ul>
+              <li>Css</li>
+              <li>Html</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
```

Comment:
- [ ] Please notice that in the technologies section on the reference. each technology tag is separated with a small thin vertical bar. please add this in order to practice matching a reference design as best as you can. 

|Your Design | Reference Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/153666092-393e4946-d43b-417b-b7cf-b7a83bb59666.png)|![image](https://user-images.githubusercontent.com/18730106/153666144-19f437dc-2567-45a6-87c4-528eb7de6559.png)|